shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -277,14 +277,6 @@ public boolean performAction(int virtualViewId, int action, Bundle arguments) {
             return false;
         }
         switch (action) {
-            case AccessibilityNodeInfo.ACTION_CLICK: {
```

Comment:
Shouldn't this stay?

Reply:
It can stay but it doesn't currently change any behavior, either

1. AccessibilityService interprets gesture -> dispatch semantics action -> framework sends tap message -> tap event fired

or

1. AccessibilityService interprets gesture -> ignored by performAction -> gesture interpreted by framework -> tap event fired

Reply:
So neither 1 or 2 ever calls performAction?

What about overlapping elements? What happens when you have accessibility focus on the bottom one and you "click" it. Will the click simulated by AccessibilitySdervice not end up on the top element if performAction is bypassed? I know that's how it worked prior to Oreo where performAction is always ignored. But I thought with Oreo this situation now behaves correctly because performAction is used and routes the click to the correct element on the bottom?

Reply:
Ahh I see what you mean.  In newer versions of Android removing this would cause all events to end up on top.  Added back the old logic here for regular clicks